utils/httpSignature.verifiers.ts

```diff
@@ -0,0 +1,60 @@
+import * as crypto from "crypto";
+import { JsonWebKey } from "crypto";
+import {
+  AlgorithmTypes,
+  verifyEcdsaSha256
+} from "@mattrglobal/http-signatures";
+
+// ----------------------
+// Custom Verifiers
+// ----------------------
+
+export const verifyRsaPssSha256 = (key: JsonWebKey) => async (
```

Comment:
Can we add some documentation and reference about this algorithm?

Reply:
Done in [b2a775e](https://github.com/pagopa/io-functions-lollipop/pull/30/commits/b2a775e13e7b0043b632e9ac979d2e16d58a6145)